src/components/InventoryGroups/Modals/AddSystemsToGroupModal.cy.js

```diff
@@ -0,0 +1,152 @@
+import { mount } from '@cypress/react';
+import {
+    checkTableHeaders,
+    MODAL,
+    ouiaId,
+    TABLE
+} from '@redhat-cloud-services/frontend-components-utilities';
+import FlagProvider from '@unleash/proxy-client-react';
+import _ from 'lodash';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import {
+    featureFlagsInterceptors,
+    groupDetailInterceptors,
+    hostsFixtures,
+    hostsInterceptors
+} from '../../../../cypress/support/interceptors';
+import {
+    selectRowN,
+    unleashDummyConfig
+} from '../../../../cypress/support/utils';
+import { getStore } from '../../../store';
+import AddSystemsToGroupModal from './AddSystemsToGroupModal';
+
+const TABLE_HEADERS = ['Name', 'Group', 'Tags', 'Update methods', 'OS', 'Last seen'];
```

Comment:
Update method, not methods
![image](https://user-images.githubusercontent.com/62722417/228529995-bea84f4b-68ff-4a5f-ac85-890abaa613eb.png)